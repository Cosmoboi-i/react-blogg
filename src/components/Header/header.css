.header {
  background: black;
  color: white;
  height: 12%;
  display: flex;
  justify-content: center;
  align-items: flex-end;
}

.content {
  width: 70%;
  height: 100%;
  justify-self: center;

  display: flex;
  flex-flow: row wrap;
  justify-content: space-between;
}

.heading {
  margin: 0;
  padding: 0.1em;
  font-weight: 100;
}

.subheading {
  font-style: italic;
}

.banner {
  align-self: center;
}

.navbar {
  display: flex;
  justify-content: space-between;

  align-self: flex-end;
  width: 40%;
  margin-bottom: 0.3em;
}