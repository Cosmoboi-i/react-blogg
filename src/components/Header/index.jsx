import React from "react";
import "./header.css";

export default function Header() {
  return (
    <div className="header">
      <div className="content">
        <div className="banner">
          <h2 className="heading">The Artifact</h2>
          <div className="subheading">Culture & Art Blog</div>
        </div>
        <div className="navbar">
          <div>Blog</div>
          <div>About</div>
          <div>Contact</div>
        </div>
      </div>
    </div>
  );
}
