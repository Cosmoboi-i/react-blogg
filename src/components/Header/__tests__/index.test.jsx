import { render } from "@testing-library/react";
import React from "react";
import Header from "..";

describe("Footer", () => {
  it("should render Footer component", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
