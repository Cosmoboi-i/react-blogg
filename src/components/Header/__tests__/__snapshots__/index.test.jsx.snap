// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer component 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="content"
    >
      <div
        class="banner"
      >
        <h2
          class="heading"
        >
          The Artifact
        </h2>
        <div
          class="subheading"
        >
          Culture & Art Blog
        </div>
      </div>
      <div
        class="navbar"
      >
        <div>
          Blog
        </div>
        <div>
          About
        </div>
        <div>
          Contact
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
